### PR TITLE
impl Send for rate limiter

### PR DESCRIPTION
**Pull request by UncP**
Signed-off-by: UncP <uncp.xu@gmail.com>

impl Send for rate limiter to fix TiKV compilation error.

### CODE REVIEW

**Comment by UncP** [comment]
@zhangjinpeng1987 PTAL

File: src/rocksdb_options.rs [hunk 1]
```diff
@@ -27,6 +27,7 @@ use slice_transform::{new_slice_transform, SliceTransform};
 use std::ffi::{CStr, CString};
 use std::mem;
 use std::path::Path;
+use std::marker::Send;
```
Comment (siddontang) [inline src/rocksdb_options.rs]: seem no need to import it here

File: src/rocksdb_options.rs [hunk 2]
```diff
@@ -153,6 +154,8 @@ pub struct RateLimiter {
     inner: *mut DBRateLimiter,
 }
 
+unsafe impl Send for RateLimiter { }
```
Comment (siddontang) [inline src/rocksdb_options.rs]: maybe we can also implement Sync here if RocksDB RaftLimiter is thread-safe.

Comment (UncP) [reply src/rocksdb_options.rs]: sure

File: src/rocksdb_options.rs (hunk 1 above)
Comment (UncP) [reply src/rocksdb_options.rs]: Fixed.

**Comment by UncP** [comment]
@siddontang @zhangjinpeng1987  PTAL

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**